handbook/company/README.md

```diff
@@ -292,6 +291,8 @@ Each group's PM works closely with engineers within their group:
 - The PM is responsible for epics. Sometimes initiatives require multiple issues that may, or may not, include multiple development groups. These initiatives are tracked as GitHub issues with the "epic" label. One PM is assigned to the epic to make sure that all issues associated with the epic (child issues) make it into a release. It's the PM's responsibility to track progress and close the epic when the initiative is complete.
 - The EM (along with engineers) defines **how** to implement that definition within the surface area of code, processes, and rituals owned by their group while serving their group’s goals and the company's goals as a whole.
 
+**Additional product design review sessions are welcomed and a top priority for the CEO to get on the calendar quickly.  Other internal meetings and blocked time can always be moved to make room.  External meetings, travel, and personal commitments cannot be moved.  To schedule, mention @Charlie Chance in the #meetings channel.**
```

Comment:
Can this be a new section called “Scheduling with CEO”?

Reply:
I've moved it to a place that makes more sense. Let me know your thoughts. 

Rationale: how the CEO interacts with the team at large and how the CEO is contacted and scheduled is a part of the culture of Fleet.